mantidproject/mantid
# resurrect the old isis refl gui

The code for the GUI was incorrectly removed when the SNS refl gui was removed.  This Pr brings the files back.

**To test:**
1. Check that the REFL, REFM and REF SF interfaces are still gone from the interfaces->reflectometry menu, and the underlying code has gone
1. Start and test the ISIS Reflectometry (old) interface.  Before this it would not start.

Fixes #21496

**Release Notes** 

*Does not need to be in the release notes.*


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
